web/testing/cypress/e2e/ID001_Create_Unique_Event.cy.js

```diff
@@ -5,7 +5,7 @@ describe("Basic app navigation", () => {
   beforeEach(() => {
     cy.viewport(1440, 900);
 
-    cy.visit("/");
```

Comment:
shouldn't this be handled by the baseURL? this way there is a single line change to switch to testing an actual URL or a different local port